models/src/main/java/com/foodie/FoodieBaseService.java

```diff
@@ -49,6 +59,18 @@ default T updateById(T entity, Key id) {
         }).orElse(null);
     }
 
+    @SuppressWarnings("unchecked")
+    default List<T> updateAllById(List<T> entities) {
```

Comment:
Your implementation of `RecordListener` class in models means this function is not needed

Reply:
I noticed it was not behaving as expected. So this is a back up